includes/wikia/ExternalUser_Wikia.php

```diff
@@ -204,15 +210,30 @@ public function getLastAuthenticationError() {
 	}
 
 	public function authenticate( $password ) {
+		// Log to check whether this is used anymore (it shouldn't be), so we can clean
+		// this up.
+		// @todo remove after the new password hashing is implemented (PLATFORM-2526)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)